CONTRIBUTING.md

```diff
@@ -36,3 +36,30 @@ Docker-compose
 - Access the running app on [local](https://0.0.0.0:8080/)
 
 -Access the Jaeger UI in [local](http://localhost:16686)
+
+## Steps to setup SonarQube locally
+
+- run `make docker`
+- Hit this [link](http://localhost:9000/) [Credentials --> Username : Admin, Password : Admin]
+- Create a local project with name as `Airport` and project key as `Airport` [Image](https://drive.google.com/file/d/1MTOFer7h9epf3UPkkE3qdIkVKaCG6bhb/view?usp=drive_link)
```

Comment:
Use relative path for the image. And store the image in this repo itself. If you store in a google drive, a new joinee will then have to request access to that file/folder, correct?

Reply:
Made changes. Moved the images to repo, and used them instead of drive.